### DISCUSSION TITLE
How to install a service disabled

### DISCUSSION THREAD

**Discussion by jchonig** [Homebrew/discussions#4187] [Writing Formulae/Casks] [answered]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

I have a formula for [conman](https://github.com/dun/conman) which installs both a client and server.  The server will fail w/o configuration so I would like to configure it not to start by default (and I personally just want the client to talk to a server on another host).  

I used to use `plist_options startup: "false"` to install the plist but not have it start.  I can not find anything in the current documentation on how to do that with `service`?  I do not see an equivalent service block method.

Thanks for your help.

Jeff

**Comment by SMillerDev** [comment, marked as answer]
Services don't start by default. You have to run `brew services start` to start them.
